.github/scripts/tkn_check_parse.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# removes `actions's` default -e flag as it breaks the logic
+set +e
```

Comment:
How would it break the logic? Can you explain? I find `+e` a bit dangerous. Say, if the script fails to install tkn, it will carry on doing stuff and probably it will pass - because the only way the script currently returns non-zero result is if the grep sees the error string.

Reply:
That's easy to solve. Just do: `tkn push bundle ... || true`

Reply:
I will try this. I thought it would just exit the before evaluate tkn exits.

Reply:
It worked. Nice tip! Love to learn these small tricks.

Reply:
Great :)